www/FabricPlugin.Answers.js

```diff
@@ -1,87 +1,62 @@
-'use strict';
 
```

Comment:
This file should be JavaScript, but it now contains es6 code. It looks like the something went wrong while transpiling the code. This breaks the entire plugin on iOS.